starship

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python
```

Comment:
still needed after @josh146's comment below?

Reply:
Yes - had a conversation about this with @josh146 offline, but basically pip will replace this with the path based on the installation environment. `scripts` works OK in this respect and no change is needed immediately.

This line is necessary for pip to translate it correctly.